addons/web_editor/static/src/js/backend.js

```diff
@@ -195,7 +195,15 @@ var FieldTextHtml = widget.extend({
     start: function () {
         var self = this;
         // Do not Forward port in >= 11.0
-        this.$formButtons = this.getParent().$buttons;
+        function _buttonsModal() {
+            // Do not Forward port in >= 11.0
+            var $modal = self.getParent().getParent();
+            if ($modal && $modal.$footer) {
+                return $modal.$footer;
+            }
+        }
+
+        this.$formButtons = this.getParent().$buttons || _buttonsModal();
```

Comment:
`start()` is called once, that is for the 1st editor that is opened.
Which means `$formButtons` is correct only the 1st time.

You should detect `$formButtons` during `initialize_content()`